dev/extra-ml-requirements.txt

```diff
@@ -40,3 +40,4 @@ mleap
 pyspark==2.4.0
 # Required by mlflow.shap
 shap
+ipython
```

Comment:
`ipython` is included as a extra dependency:

https://github.com/slundberg/shap/blob/54e6f0f10aa0442734aade259fc9712c4bf16537/setup.py#L183-L187

```python
    extras_require = {
        'plots': [
            'matplotlib',
            'ipython'
        ],
```

`matplotlib` is wrapped by try-except to prevent an import error:

https://github.com/slundberg/shap/blob/54e6f0f10aa0442734aade259fc9712c4bf16537/shap/plots/_image.py#L8

```python
try:
    import matplotlib.pyplot as pl
except ImportError:
    warnings.warn("matplotlib could not be loaded!")
    pass
```
but `ipython` is not.

```

```

Reply:
```
pip install 'shap[plots]'
```

should work too but this installs `matplotlib` too

Reply:
Filed a PR in the shap repo: https://github.com/slundberg/shap/pull/1749

I'll insert a comment on this.